### DISCUSSION TITLE
MY REACT PAGE IS SHOWING BLANK PAGE WHEN I GO LIVE ON GITHUB

### DISCUSSION THREAD

**Discussion by Encoded99** [community/community#62786] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello GitHub community,

I am facing some issues with deploying my React app on GitHub Pages. The app works fine locally, but when I access it on the live URL, I encounter two main problems:

The page appears blank when accessed on GitHub Pages or via VS Code Live Server on my phone.
In the browser console, I'm seeing multiple "Failed to load resource: the server responded with a status of 404 ()" errors for the JavaScript, CSS, and "manifest.json" files.
I have already taken the following steps to troubleshoot:

Ensured that the "homepage" field in "package.json" is correctly set to the GitHub Pages URL.
Verified that the favicon, JavaScript, CSS, and "manifest.json" files are present in the "public" folder and correctly named.
Cleared my browser cache to rule out any caching issues.
Despite these efforts, the issues persist, and I'm unable to figure out the root cause of the problems.

Could someone please assist me in identifying what might be causing these issues and how I can resolve them? I would greatly appreciate any insights, suggestions, or steps to investigate further.

Thank you in advance for your help!

**Comment by caseybarajas** [comment]
Hey Encoded99, 

This may seem like a stupid answer, but I don't believe that you can host a reactive site on GitHub Pages. Pages is only for static websites, not reactive Node.js websites using React. 

Hope this Helps
-Casey Barajas

**Comment by alriyadsouza** [comment]
1) Double-Check homepage Configuration
2) Check Relative Paths
3) check your GitHub repository to see if all the required files are correctly pushed to the repository, especially the "build" folder if you're using it.
4) GitHub Pages might cache assets. After making changes, try appending a query parameter like ?v=1 to your asset URLs to force a cache refresh.

**Reply by Encoded99** [reply]
I have done all that I think I even created a fresh new repository and follow the tutorial video strictly which everything seem right but I'm still getting page not found

**Comment by 2-GO** [comment]
Hello @Encoded99 ,

This website [](https://www.c-sharpcorner.com/article/how-to-deploy-react-application-on-github-pages/) helped me with deploying my react app on gh-pages; creating a gh-pages branch, running deploy, and committing the changes to gh-pages.  Maybe it could provide some useful information to you as well.

**Comment by Encoded99** [comment]
Alright thanks

**Comment by Murat-Karakaya** [comment]
I am assuming the problimatic repo is you "mix" repository.
In your gh-pages branch, change the change the index.htm file:


``<link rel="icon" href="/mix/favicon.ico"/>`` to ``<link rel="icon" href="./favicon.ico"/>``
``<link rel="apple-touch-icon" href="/mix/logo192.png"/>`` to ``<link rel="apple-touch-icon" href="./logo192.png"/>``
``<script defer="defer" src="/mix/static/js/main.7902cbbc.js">`` to ``<script defer="defer" src="./static/js/main.7902cbbc.js">``
.
.
.
and so on.


 What I am doing here is changing the ``src="/mix/example"`` to ``src="./example"`` . Because when your site is deployed. Your index.html file doesn't knows what "mix" is. So it can't reach to your logo, javascript, css. Thats why it gives you an error and a blank page.

If it doesn't works, please let me know.

**Reply by Murat-Karakaya** [reply]
I know I didn't explained the problem properly. So if it sounds confusing, please let me know
